src/components/NavLinks.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+const NavLinks = () => {
+  const navStyle = {
+    display: 'flex',
+    listStyleType: 'none',
+  };
+  const navLink = {
+    textDecoration: 'none',
+    width: '3.688rem',
+    height: '1rem',
+    margin: '1.125rem 2.063rem 0.688rem 0',
+    fontSize: '1rem',
+    letterSpacing: '1.9px',
+    color: '#0290ff',
+  };
+
+  const line = {
+    borderRight: '2px solid black',
+    marginRight: '2.563rem',
+  };
+  return (
+    <nav id="navbar-container">
+      <ul style={navStyle}>
+        <li className="nav-link">
+          <NavLink to="/" style={navLink} id="navLink">Rockets</NavLink>
```

Comment:
- [ ] As required. I kindly suggest that your current active page is indicated with an underline using the `NavLink`.



| Your Design | Figma Design |
| ------------- | ------------- |
| <img width="278" alt="image" src="https://user-images.githubusercontent.com/52934128/196006558-fd873bac-2c59-4053-b2f5-97402a0284b3.png">  | <img width="388" alt="image" src="https://user-images.githubusercontent.com/52934128/196004277-be9dd8dc-4e01-4f47-a9a5-cf16627748d6.png">  |


## EXAMPLE
```
 <ul>
        <li>
          <NavLink exact activeClassName="active" to="/">
            Home
          </NavLink>
        </li>
        <li>
          <NavLink activeClassName="active" to="/users">
            Users
          </NavLink>
        </li>
        <li>
          <NavLink activeClassName="active" to="/contact">
            Contact
          </NavLink>
        </li>
      </ul>
```

```
index.css

.active{
  color:red;
}

```

Please check out [this article](https://reactgo.com/reactrouter/navlink/#:~:text=What%20is%20the%20difference%20between,Let's%20style%20them%20using%20NavLink.) for more details.